apps/api/src/app/shared/content/fandom-tag.interface.ts

```diff
@@ -0,0 +1,5 @@
+import { CreateFandomTag, FandomTagModel } from "@dragonfish/shared/models/content";
+
+export interface IFandomTag {
```

Comment:
So I know that you're following the pattern here, but I don't love the pattern: something named `IFandomTag` makes me think that this is an individual _tag_, not a service (or interface in this case) that _manages_ tags.